### PR TITLE
Update FTS FAQs with search scoring information

### PR DESCRIPTION
**Pull request by sreekanth-cb**

### CODE REVIEW

**Review by steveyen — APPROVED** [review]

File: modules/fts/pages/fts-troubleshooting.adoc [hunk 1]
```diff
@@ -160,3 +160,52 @@ These are scorch specifc commands ...
...
 Invoking the commands above with --help will highlight more information and further sub commands available for each.
+
+*Q: How does FTS scores documents?*
+
+FTS's internal text indexing library(bleve) uses a slightly modified version of standard tf-idf scoring. This improvisation is done to normalise the score by various relevant factors. The search scoring happens at the query time.
+
+When bleve scores a document - it sort of sums a set of sub scores to reach the final score. Scores across different searches are not directly comparable as the search query is also an input factor to the scoring function. The more conjuncts/disjuncts/sub clauses your query has, the more it will influence the scoring.
+The score of a particular hit is not absolute, meaning that it can only be used as a comparison to the highest score from the same search result. There isn't a pre-defined range for valid scores. 
+
+To summarise the scoring function in a comprehensive way,
```
Comment (amarantha-k) [inline modules/fts/pages/fts-troubleshooting.adoc]: incomplete sentence?

File: modules/fts/pages/fts-troubleshooting.adoc [hunk 2]
```diff
@@ -160,3 +160,52 @@ These are scorch specifc commands ...
  ** command-line tool to interact with a zap file
 
 Invoking the commands above with --help will highlight more information and further sub commands available for each.
+
+*Q: How does FTS scores documents?*
```
Comment (amarantha-k) [inline modules/fts/pages/fts-troubleshooting.adoc]: How does the Search service (FTS) score documents?

File: modules/fts/pages/fts-troubleshooting.adoc [hunk 3]
```diff
@@ -160,3 +160,52 @@ These are scorch specifc commands ...
...
+
+FTS's internal text indexing library(bleve) uses a slightly modified version of standard tf-idf scoring. This improvisation is done to normalise the score by various relevant factors. The search scoring happens at the query time.
```
Comment (amarantha-k) [inline modules/fts/pages/fts-troubleshooting.adoc]: Update last sentence:
The search scoring happens at query time.

File: modules/fts/pages/fts-troubleshooting.adoc [hunk 4]
```diff
@@ -160,3 +160,52 @@ These are scorch specifc commands ...
...
+fieldNorm(f) = 1 / SQROOT(nTokens(f))
+idf(f, ti) = 1 + LN(|Docs| / (1 + FREQ(ti, FIELDNAME(f), Docs)))
+Docs = a set of all indexed documents
+----
+
+where SQROOT, SUM, and LN denote standard mathematical functions. Auxiliary functions are:
+
+* *coord(q, f)*  — is a dampening factor defined as a ratio of query tokens that are found in the given field, and the total number of tokens in a query.
+* *tw(ti, q, f)*  — *ti* ’s term weight is the product of  *ti* ’s query weight and ti’s field weight.
+* *queryWeight(q, ti)*  —  *ti* ’s query weight (wrt to  *q* ) is the product of its [inverse document frequency](https://en.wikipedia.org/wiki/Tf%E2%80%93idf) (see  *idf*  below) and its [boosting factor](http://www.blevesearch.com/docs/Query-String-Query/).
```
Comment (amarantha-k) [inline modules/fts/pages/fts-troubleshooting.adoc]: INFO: External links are in the following format: URL[Link text to display] 
For example, here are a few that need updating. Please update all the other links too.
- https://en.wikipedia.org/wiki/Tf%E2%80%93idf[inverse document frequency]
- http://www.blevesearch.com/docs/Query-String-Query/[boosting factor]

**Review by amarantha-k — COMMENTED** [review]
@sreekanth-cb Please take a look at my review comments, esp around links. Once updated, i'll be happy to merge the changes.

**Comment by sreekanth-cb** [comment]
thanks @amarantha-k , done!

**Review by amarantha-k — APPROVED** [review]

**Pull request merged**